src/types/markers.js

```diff
@@ -113,13 +141,18 @@ export type MarkerSchema = {|
         label?: string,
         format: MarkerFormatType,
         searchable?: boolean,
+        // hidden in the side bar and tooltips?
+        isHidden?: boolean,
```

Comment:
I think the name for this property isn't really good, it's not super clear what this is for.
What about: 
```js
  // Should we display this property in the sidebar and tooltips?
  // The default is yes.
  displayInDetails?: boolean
```

Initially I was wondering that we may want an array of enum similar to `display` instead. For example:
```js
  // Default is ['sidebar', 'tooltip'], specify null if you want to avoid displaying anywhere
  // Currently only the value `null` is supported
  display?: Array<'sidebar' | 'tooltip'> | null
```
But this seems overkill.